content/javascript/concepts/dates/terms/toDateString/toDateString.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'toDateString()'
+Description: 'returns a string representing the date portion of a Date instance' 
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`toDateString()`** method of Date instances returns the date portion of a Date object as a string.
+
+## Syntax
+```pseudo
+const d = new Date('January 1, 2000 00:00')
+d.toDateString()
+```
+
+## Example
+```js
```

Comment:
add a small introductory sentence above this code snippet which can act as a path for author to get ready and what to expect from the code snippet below.